### ISSUE TITLE
[Task]: Replace Recommended Add-on Icon on AMO

### ISSUE THREAD

**Issue by abyrne-moz** [mozilla/addons#15159]
### Description
The current Recommended icon on AMO is not accessible due to the low contrast.  We would like to replace it with a new version across all of AMO that better aligns to our brand and A11y guidelines

### Acceptance Criteria
```
  ### Milestones/checkpoints
  - [ ] The old icon has been replaced with the new in AMO
  - [ ] Update [support articles](https://support.mozilla.org/en-US/kb/add-on-badges) with new logo
  - [ ] Update [ExtensionWorkshop](https://extensionworkshop.com/documentation/publish/recommended-extensions/)
with new logo
```

### Checks
* [x] If I have identified that the work is specific to a repository, I have removed "repository:addons-server" or "repository:addons-frontend"

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/AMOENG-1220)
┆Attachments: <a href="https://mozilla-hub.atlassian.net/rest/api/2/attachment/content/73904">competitiveness-20.svg</a> | <a href="https://mozilla-hub.atlassian.net/rest/api/2/attachment/content/75023">image-20241220-140459.png</a>

**Comment by abyrne-moz** [comment]
Nicole, can you please attach or link to the assets we need?

**Comment by willdurand** [comment]
Please update SUMO when this is done: https://support.mozilla.org/en-US/kb/add-on-badges :)

**Comment by willdurand** [comment]
This is how it should look like, it seems:

![Image](https://github.com/user-attachments/assets/337da72e-b8e2-43f8-afc8-1bb580944269)

**Comment by topotropic** [comment]
I'm looking at refreshing the listing page and as part of that also at the badges, please hold off and I'll update the tickets when I'm done. Thanks!

**Comment by topotropic** [comment]
ok, here's the update: 

![Image](https://github.com/user-attachments/assets/e5c66406-019b-4579-af5e-112aeca0a7e1)
![Image](https://github.com/user-attachments/assets/06f19376-4510-4167-8c15-1ce6e601e6a0)

**border:** 
$color-light-gray-40
#CFCFD8

  **text:** 
 $color-dark-gray-05
#5B5B66

**icon:**  
$color-orange-50
#ff7139 
16x16px

Colors are from protocol https://protocol.mozilla.org/docs/fundamentals/color -- reach out if you have any feedback or questions. TY!

**Assets:** 
![Image](https://github.com/user-attachments/assets/debca235-e0c9-4c7f-833f-02658453c2c3)

![Image](https://github.com/user-attachments/assets/292b2d5f-f1d9-4f4a-8835-73354327068d)

**Comment by alexandruschek** [comment]
Hello,

I checked this on -DEV:
Recommended Icon now looks like this:
<img width="423" alt="Image" src="https://github.com/user-attachments/assets/7de80b62-514b-4c5e-ba3f-7b958ab18bd7" />
By Firefox looks like this:
<img width="423" alt="Image" src="https://github.com/user-attachments/assets/07634fff-02df-4545-b7e4-4ea45ab27e50" />
Color: #5b5b66

Thanks!
